velox/functions/lib/string/tests/StringImplTest.cpp

```diff
@@ -431,6 +431,20 @@ TEST_F(StringImplTest, getByteRange) {
     EXPECT_EQ(expectedStartByteIndex, range.first);
     EXPECT_EQ(expectedEndByteIndex, range.second);
   }
+
+  // Test bad unicode strings.
+
+  // This exercises bad unicode byte in determining startByteIndex.
+  std::string badUnicode = "aa\xff ";
+  auto range = getByteRange<false>(badUnicode.data(), 4, 3);
```

Comment:
the string length above is <6 in size (its 5)? would not this potentially seg fault during the call?
shall we add a space at the end of badUnicode?

Reply:
There is a space at end of bad unicode (?)

Reply:
what is the length in bytes for badUnicode? is it addressable at index 5?